### PR TITLE
Ping Command

### PR DESCRIPTION
**Pull request by shashankv327**

### CODE REVIEW

File: src/main/scala/Commands/Ping/Ping.scala [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  override val aliases: Seq[String] = Seq("ping")
+  override val name: String = "Ping"
+
+  override def getSubCommand(event: MessageReceivedEvent): SubCommand = {
+    Ping
+  }
+
+  override def execute(event: MessageReceivedEvent, transactor: Resource[IO, HikariTransactor[IO]]): Unit = {
+    event.getChannel.sendMessage(
+      "Pong \n").queue()
```
Comment (dhyanpatel) [inline src/main/scala/Commands/Ping/Ping.scala]: This is formatted weirdly, put it all in one line.

Comment (dhyanpatel) [inline src/main/scala/Commands/Ping/Ping.scala]: Why the `<space>\n`?

**Review by dhyanpatel — CHANGES_REQUESTED** [review]

File: src/main/scala/Commands/Ping/Ping.scala [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+object Ping extends Command with SubCommand {
+  override val aliases: Seq[String] = Seq("ping")
+  override val name: String = "Ping"
+
+  override def getSubCommand(event: MessageReceivedEvent): SubCommand = {
+    Ping
+  }
+
+  override def execute(event: MessageReceivedEvent, transactor: Resource[IO, HikariTransactor[IO]]): Unit = {
+    event.getChannel.sendMessage("Pong\n").queue()
```
Comment (dhyanpatel) [inline src/main/scala/Commands/Ping/Ping.scala]: Why the `\n`?

**Review by dhyanpatel — CHANGES_REQUESTED** [review]

**Review by dhyanpatel — APPROVED** [review]

**Review by Nikhil-Pr — APPROVED** [review]
good job bb proud of you v nice 👍

**Pull request merged**